cmd/http-check/main.go

```diff
@@ -71,12 +71,12 @@ func main() {
 		if err != nil {
 			log.Fatalln("error when reporting to kuberhealthy:", err.Error())
 		}
-		log.Println("Successfully reported to Kuberhealthy of failure")
+		log.Infoln("Successfully reported to Kuberhealthy of failure")
 	}
 
 	err := kh.ReportSuccess()
 	if err != nil {
-		log.Fatalln("error when reporting to kuberhealthy:", err.Error())
+		log.Infoln("error when reporting to kuberhealthy:", err.Error())
```

Comment:
This used to Fatal, but now it only calls info, and then keeps executing.

Reply:
I have corrected this back to log.Fatalln